app/components/list-item/index.tsx

```diff
@@ -0,0 +1,27 @@
+import { ListItemType } from '@/app/app-types'
+
+type ListItemProps = {
+  items: ListItemType[]
+}
+
+const ListItem = (props: ListItemProps) => {
+  return (
+    <ul data-testid="list-item-wrapper" className="pl-4">
+      {props.items.map((listItem) => {
```

Comment:
Always add null check before accessing the list

Reply:
Added the null check.